src/main/java/edit_user_public_profile_screens/editUserPublicProfileController.java

```diff
@@ -0,0 +1,20 @@
+package edit_user_public_profile_screens;
```

Comment:
This is a super minor thing but remember to capitalize your class names!

Reply:
I also wasn't sure about naming conventions between packages and files so that might be another thing we need to discuss!